File: BMICalculator/app/src/main/java/com/example/bmicalculator/MainActivity.java
package com.example.bmicalculator;

import androidx.appcompat.app.AppCompatActivity;

import android.annotation.SuppressLint;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    Button calculate;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.activity_main);

        calculate = findViewById(R.id.calculate);

        calculate.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent myintent = new Intent(MainActivity.this,Calculate.class);
                startActivity(myintent);
            }
        });


    }
}

File: First_Class/app/src/main/java/com/rumanweb/first_class/MainActivity.kt
package com.rumanweb.first_class

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.webkit.WebView
import android.webkit.WebViewClient
import android.widget.Button
import android.widget.TextView
import android.widget.Toast
import kotlinx.coroutines.newSingleThreadContext

class MainActivity : AppCompatActivity() {
    private lateinit var webView: WebView
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        findViewById<TextView>(R.id.hire).setOnClickListener {

            Toast.makeText(this, "I am hired for your project, 20$ has been deducted from your account.",Toast.LENGTH_LONG).show()


        }
        findViewById<Button>(R.id.contact_me).setOnClickListener {
            startActivity(Intent(this, another_activity:: class.java))

        }

    }
}